src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+@Override
+public void doWork() {
+    System.out.println("Bulldozer do work");
+}
+@Override
+public void stopWork() {
+    System.out.println("Bulldozer stoped work");
```

Comment:
There is a typo in the string "Bulldozer stoped work". It should be "Bulldozer stopped work".